hwdb.d/70-multimedia.hwdb

```diff
@@ -22,6 +22,29 @@ usb:v2982p1967*
 usb:v1157p300*
  ID_MULTIMEDIA_CONTROLLER=1
 
+################
+# Elgato
+################
+# Stream Deck Original (gen 1)
+usb:v0FD9p0060*
+ ID_MULTIMEDIA_CONTROLLER=1
```

Comment:
Add a link to the "Stream Deck" product page on Elgato's website, or a Wikipedia link, in the commit message.

Reply:
Added a link to https://www.elgato.com/en/stream-deck in the commit message